client/src/components/Task/PlayPauseButton/PlayPauseButton.js

```diff
@@ -0,0 +1,54 @@
+import "./PlayPauseButton.scss";
+import React from "react";
+import IconButton from "@material-ui/core/IconButton";
+import PauseIcon from '@material-ui/icons/Pause';
+import PlayArrowIcon from '@material-ui/icons/PlayArrow';
+import Box from "@material-ui/core/Box";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import Fade from "@material-ui/core/Fade";
+
+export default function PlayPauseButton(props) {
+    const [started, toggleStarted] = React.useState(false);
+    const [progress, setProgress] = React.useState(0);
+
+    React.useEffect(() => {
```

Comment:
correct me if im wrong, so this one will keep running the moment we access the page? Why not set it to run/stop only when playClick is triggered?